src/test/groovy/com/michalpu/PokeRest/pokemon/infrastructure/api/PokemonEndpointTest.groovy

```diff
@@ -26,8 +29,26 @@ class PokemonEndpointTest extends BaseIntegrationTest {
         then:
         with(pokemon) {
             id == 4
-            name.equalsIgnoreCase("charmander")
-            weight == 85
+            name.equalsIgnoreCase("charmander");
+            weight == 86
         }
     }
+
+
+    void 'should return types vulnerable to pokemons type'  () {
+
+        when:
+        def typeRelations = restTemplate.getForObject(localUrl("/pokemons/charmander/type-relations"),
+                                                                TypeRelations.class)
+        then:
+        with(typeRelations){
+            noDamageTo.isEmpty()
+            noDamageFrom.isEmpty()
+            doubleDamageFrom.containsAll(Arrays.asList("ground", "rock", "water"))
```

Comment:
You can create a list in Groovy by using [ ], for example `["one", "two", "three"]`